docs/includes/file-tutorial-excel-custom-functions.md

```diff
@@ -43,30 +43,43 @@ You’ll begin this tutorial by using the Yo Office generator to create the file
     * Choose a script type: `JavaScript`
     * What do you want to name your add-in? `stock-ticker`
 
-    ![Yo Office bash prompts for custom functions](../images/yo-office-cfs-stock-ticker-3.png)
+    ![Yo Office bash prompts for custom functions](../images/12-10-fork-cf-pic.jpg)
 
     After you complete the wizard, the generator will create the project files and install supporting Node components. The project files come from the [Excel-Custom-Functions](https://github.com/OfficeDev/Excel-Custom-Functions) GitHub repository.
 
 2. Navigate to the project folder.
 
-    ```bash
+    ```
     cd stock-ticker
     ```
 
-3. Start the local web server.
+3. Trust the self-signed certificate that is needed to run this project. For detailed instructions for either Windows or Mac, see [Adding Self Signed Certificates as Trusted Root Certificate](https://github.com/OfficeDev/generator-office/blob/master/src/docs/ssl.md).  
+
+4. Build the project.
+
+    ```
+    npm run build-dev
+    ```
+    
+    After running this, you will see a readout in your command prompt about the build process. Note that you will be using this command later to re-build whenever you make changes to your functions files.
+
+5. Start the local web server, which runs in Node.js. 
 
     * If you'll be using Excel for Windows to test your custom functions, run the following command to start the local web server, launch Excel, and sideload the add-in:
 
```

Comment:
If using web and to debug...
`npm run start-web` and use the browser to debug

if using windows and to debug..

`npm run start-desktop`
open vs code
debug steps